examples/VisualNovel/Main.gd

```diff
@@ -56,16 +63,26 @@ func set_background(bg_path: String, smooth: float = 0.5, speed: float = 1.0):
 
 func _on_EventManager_event_started(event: Event):
 	current_event = event
+	if event is EventDialogText:
+		current_character = event.character
+		if current_character:
+			var node = dialog_node.portrait_manager.portraits[current_character]
```

Comment:
Woops, I forgot to open the issue to add in PortraitManager a method to get the node related to the character

Reply:
https://github.com/AnidemDex/Godot-DialogPlugin/issues/73